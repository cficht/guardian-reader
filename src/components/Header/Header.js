import React from 'react';
import Logo from '../../assets/guardian_logo.png';
import './Header.scss';

export default function Header() {
  return (
    <header>
      {/* <h1>Guardian Reader</h1> */}
      <img src={Logo} alt="Guardian Logo"/>
    </header>
  );
}
