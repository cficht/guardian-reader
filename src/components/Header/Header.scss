$primary-color: #052962;
$secondary-color: #FFFFFF;
$third-color: whitesmoke;
$fourth-color: #00417B;
$fifth-color: #B2C9D9;

header {
  display: flex;
  justify-content: center;
  background-color: $primary-color;

  img {
    height: 100%;
    width: 50%;
    padding: 20px 0px 0px 0px;
  }
}
