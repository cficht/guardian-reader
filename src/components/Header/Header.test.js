/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

test('renders Header.js correctly', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
