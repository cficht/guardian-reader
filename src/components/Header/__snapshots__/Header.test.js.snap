// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header.js correctly 1`] = `
<header>
  <img
    alt="Guardian Logo"
    src="guardian_logo.png"
  />
</header>
`;
